content/support.rst

```diff
@@ -0,0 +1,193 @@
+is via the Google Group email.
+Each message to the Support email automatically creates a helpdesk ticket.
+
+
+Template: Obtaining the installation detail
+-------------------------------------------
+
+The following is a template to obtain the installation detail.
+Please modify the template below to suit.
+
+.. sourcecode:: rst
+
+	Can you please provide us with the following details about the system
+	used for running the product:
+
+	* Product version?
+
+	* Operating system name and version?
+
+	* File transfer protocol(s) used?
+
+	* Please send through a time-stamped version of the server-side log, and
+	   if relevant, the corresponding client-side log.
+
+	* Other information (if known).
+
+	* Please advise brief details of any: integration, software, resilience / redundancy /
+	high availability architecture, security or other technologies that interact with the
+	product.
+	e.g. Firewalls, Proxy, LDAP, Active Directory, Single Sign On, Management software etc.
+
+	Before releasing a new version, we run an automated test suite to validate new features
+	and check for regressions.
+	
+	By knowing details about your installations, we can configure the test suite to run
+	against a similar setup.
+
+	This information will not be shared with any other third party and will be used
+	solely for improving further support for you and development of the product.
+
+
+Using the installation details
+------------------------------
+
+Once the installation details are provided, recreate the issue in a similar
+environment.
+
+This should be created via a dedicated VM for testing / support activities.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat with our customers
+or possible customers.
+
+Each chat transcript is automatically forwarded to the Support email list.
+
+A chat can be transferred and other actions can be taken with the chat via
+Olark `commands <https://www.olark.com/help/commands>`_.
+
+
+Licenses and support contract
+=============================
+
+See the `Support options page <https://www.sftpplus.com/support/options.html>`_
+which describes the options available.
+
+If a support case goes beyond the standard option (such as a new
+product feature), the Sales team will determine the quote.
+
+See the `Life cycle page <https://www.sftpplus.com/product/life-cycle.html>`_ 
+for details about the product life cycle.
+
+If a feature request is obtained, check that it is included in the
+`Product roadmap <https://www.sftpplus.com/product/roadmap.html>`_ 
+
+
+Support Case Management
+=======================
+
+
+Working on a support case
+-------------------------
+
+Support cases can be received either through the online contact form,
+direct email or phone call.
+
+Email and support forms are the preferred methods.
+When we have a phone call, we request the customer at the beginning to do a
+follow up over email and confirm in writing.
```

Comment:
And if the customer is not doing it, we should follow up with an email, summarizing the phone call and ask the customer to confirm that this is indeed what was discussed